iot-hub/Samples/service/ServiceClientSample/ServiceClientSample.cs

```diff
@@ -50,13 +54,50 @@ public async Task RunSampleAsync(TimeSpan runningTime)
 
         }
 
+        private async Task ReceiveMessageFeedbacksAsync(CancellationToken token)
+        {
+            _logger.LogInformation("Starting to listen to feedback messages");

```

Comment:
nit: we could use a message here informing the users that in order to see the message feedback receiver in action the device-side counterpart should be active.